lib/is.js

```diff
@@ -0,0 +1,100 @@
+'use strict';
+
+// Ref; https://github.com/OpenObservability/OpenMetrics/blob/1dcecc569894abef6b67037b5e14f1813c7b84d7/metric_exposition_format.md
+const REGEX_NAME = /^[a-zA-Z_][a-zA-Z0-9_]*$/;
+const REGEX_LABEL = /^[a-zA-Z0-9_]*$/;
+
+const isString = (value) => {
```

Comment:
Is V8 able to optimise this approach as well as a named function?

Reply:
Yes, according to [this](https://docs.google.com/document/d/1EA9EbfnydAmmU_lM8R_uEMQ-U_v4l9zulePSBkeYWmY/edit#heading=h.q3863ga20xbb) all optimizations V8 does for normal functions also apply to arrow functions 🙂

Reply:
The important part perf vise is that its assigned to a constant. NOTE: `module.exports.fooBar` are not constants.